photocolle-sdk/Makefile

```diff
@@ -20,7 +20,10 @@ doc:
 	zip target/$(SDKNAME)-iOS-$(VERSION)_API_doc.zip -r $(DOCOUTPUTDIR)
 
 framework:
-	sh photocolle-sdk.framework/generate_framework.sh
+	(chmod 755 ios-build-framework-script.sh; \
```

Comment:
Is `chmod` necessary in this line?
I think committing `ios-build-framework-script.sh` as 755 is fine.
What do you think?